types/three/examples/jsm/nodes/core/CodeNode.d.ts

```diff
@@ -0,0 +1,12 @@
+import { NodeTypeOption } from './constants';
+import Node from './Node';
+import NodeBuilder from './NodeBuilder';
+
+export default class CodeNode extends Node {
+    isCodeNode: true;
+    code: string;
+    constructor(code: string, nodeType?: NodeTypeOption);
+
+    setIncludes(includes: any[]): this;
+    getIncludes(builder: NodeBuilder): any[];
```

Comment:
this doesn't actually require a bulder. I wonder if `unknown` would be better as opposed to `any` CC @Methuselah96 for his opinion.

Reply:
Was replaced by a new type `CodeNodeInclude`